features/lwipstack/lwipopts.h

```diff
@@ -339,7 +339,9 @@
 #define LWIP_ETHERNET               0
 #endif // MBED_CONF_LWIP_ETHERNET_ENABLED
 
-#define LWIP_L3IP   				0
+#define LWIP_L3IP                   0
+//Maximum size of network interface name
+#define INTERFACE_NAME_SIZE 6
```

Comment:
spaces

Reply:
should this be defined to be NSAPI_INTERFACE_NAME_MAX_SIZE

Reply:
INTERFACE_NAME_SIZE  renamed to  INTERFACE_NAME_MAX_SIZE 
Modified lwipopts.h 
#define INTERFACE_NAME_MAX_SIZE NSAPI_INTERFACE_NAME_MAX_SIZE  to avoid including nsapi_types.h  in lwip   dns and netif